### ISSUE TITLE
Can't install tab center when internet security suites are enabled

### ISSUE THREAD

**Issue by clouserw** [bwinton/TabCenter#727]
[kinda cloned from https://github.com/mozilla/activity-stream/issues/1509]

Firefox does some special checks when updating add-ons to make sure the new version is the right one. One of these is checking the update against a whitelist of certs. Many internet security suites inject their own root certs so they can MITM SSL traffic which breaks this update check.

Until this gets addressed further up the chain, our best bet is to bypass the cert check by passing a sha hash of the .xpi through a redirect to install. (The reason this check exists is the idea that you want to host the redirect on a secure site, but the .xpi itself on an insecure CDN).

So, long story short, to fix this (the following links are 404s at this point):

1) Send me your current .xpi file.  I can host it on our S3 bucket and put the right redirect jazz around it to make this work.
2) Create a new version of tab center with the update URL pointing to https://testpilot.firefox.com/files/tabcenter/updates.json
3) Change any install links you have (including the one on testpilot.firefox.com) to https://testpilot.firefox.com/files/tabcenter/latest

Some background:

https://bugzilla.mozilla.org/show_bug.cgi?id=1308251
mozilla/testpilot#1474
mozilla/testpilot#1537

**Comment by wresuolc** [comment]
Install Link: https://testpilot.firefox.com/files/tabcentertest1@mozilla.com/latest
Update URL: https://testpilot.firefox.com/files/tabcentertest1@mozilla.com/updates.json

**Comment by bwinton** [comment]
I need to:
- [ ] fix those,
- [ ] generate a new version,
- [ ] get it signed,
- [ ] send it to Wil.
